admin/src/js/controllers/message-queue.js

```diff
@@ -0,0 +1,102 @@
+var moment = require('moment');
+
+angular.module('controllers').controller('MessageQueueCtrl',
+  function(
+    $log,
+    $q,
+    $scope,
+    $state,
+    MessageQueue,
+    Settings
+  ) {
+
+    'use strict';
+    'ngInject';
+
+    var delayInterval = 5 * 60 * 1000, // 5 minutes
+        tab = $state.current.data.tab,
+        descending = $state.current.data.descending;
+
+    var normalizePage = function(page) {
+      page = parseInt(page);
+      if (isNaN(page) || page <= 0) {
+        return 1;
+      }
+      return page;
+    };
+
+    $scope.pagination = {
+      page: normalizePage($state.params.page),
+      perPage: 25,
+      pages: 0
+    };
+    $scope.displayLastUpdated = tab !== 'scheduled';
+    $scope.loading = true;
+
+    var formatMessages = function(messages) {
+      if (tab === 'due') {
+        var transitionalStates = [ 'pending', 'forwarded-to-gateway', 'forwarded-by-gateway', 'received', 'sent' ],
```

Comment:
Pull `transitionalStates` out with the `delayInterval` constant so it's clearer it's a constant.